lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidator.test.js

```diff
@@ -0,0 +1,404 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../../tests/utils/run-serverless');
+const fixtures = require('../../../../../../../../tests/fixtures');
+
+const expect = chai.expect;
+chai.use(require('chai-as-promised'));
+
+describe('#compileRequestValidators()', () => {
+  let globalServerless;
+
+  before(() => {
+    globalServerless = runServerless({
```

Comment:
Let's rename it to `serverlessInstance`, also we should store resolved instance and not _promise_ that eventually resolves the instance